tests/base.py

```diff
@@ -798,6 +813,14 @@ def getPull(self, owner, project, number):
         }
         return data
 
+    def getPullBySha(self, sha):
+        prs = list(set([p for p in self.pull_requests if sha == p.head_sha]))
+        if len(prs) > 1:
+            raise Exception('Multiple pulls found with head sha: %s' % sha)
```

Comment:
Im not sure if we want to raise an error here or what? Is this case even possible?